### PR TITLE
Add java presubmit

### PR DESCRIPTION
**Pull request by jaxesn**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templater/jobs/presubmit/eks-distro-build-tooling/eks-distro-base-presubmits-java-2022.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+jobName: eks-distro-base-tooling-presubmits-csi-2022
```
Comment (abhay-krishna) [inline templater/jobs/presubmit/eks-distro-build-tooling/eks-distro-base-presubmits-java-2022.yaml:1]: This should be change to a Java-based name. Same for below

Comment (jaxesn) [reply templater/jobs/presubmit/eks-distro-build-tooling/eks-distro-base-presubmits-java-2022.yaml:1]: ah yeah

File: templater/jobs/presubmit/eks-distro-build-tooling/eks-distro-base-presubmits-java-2022.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+runIfChanged: eks-distro-base/.*|scripts/setup_public_ecr_push.sh
+imageBuild: true
+localRegistry: true
+commands:
+- export DATE_EPOCH=$(date "+%F-%s")
+- make minimal-images-base-java17 -C eks-distro-base IMAGE_TAG=${DATE_EPOCH}.${AL_TAG}
```
Comment (abhay-krishna) [inline templater/jobs/presubmit/eks-distro-build-tooling/eks-distro-base-presubmits-java-2022.yaml:7]: does it need the 17 prefix?

Comment (abhay-krishna) [reply templater/jobs/presubmit/eks-distro-build-tooling/eks-distro-base-presubmits-java-2022.yaml:7]: i guess it's like the Python one in this respect.

**Pull request merged**